### PR TITLE
Add google cdn

### PR DESCRIPTION
**Pull request by barakplasma**
Added Google CDN
You can verify that the response header "Via: 1.1 google" is their header [here](https://cloud.google.com/cdn/docs/troubleshooting)

### CODE REVIEW

File: agent/browser/ie/pagetest/cdn.h [hunk 1]
```diff
@@ -156,6 +156,7 @@ CDN_PROVIDER_HEADER cdnHeaderList[] = {
 	{"server", "gws", _T("Google")},
 	{"server", "GSE", _T("Google")},
 	{"server", "Golfe2", _T("Google")},
+	{"Via", "1.1 google", "Google"},
```
Comment (pmeenan) [inline agent/browser/ie/pagetest/cdn.h]: Slight tweak, this one needs to be _T("Google")

**Comment by barakplasma** [comment]
I removed '1.1' from both and added _T .

File: agent/browser/ie/pagetest/cdn.h (hunk 1 above)
Comment (barakplasma) [inline agent/browser/ie/pagetest/cdn.h]: Done in a later commit

Comment (barakplasma) [inline agent/browser/ie/pagetest/cdn.h]: Done in a later commit

File: agent/browser/ie/pagetest/cdn.h [hunk 2]
```diff
@@ -156,6 +156,7 @@ CDN_PROVIDER_HEADER cdnHeaderList[] = {
 	{"server", "gws", _T("Google")},
 	{"server", "GSE", _T("Google")},
 	{"server", "Golfe2", _T("Google")},
+	{"Via", "google", _T"Google"},
```
Comment (pmeenan) [inline agent/browser/ie/pagetest/cdn.h]: Sorry again, look at the lines above and below - _T("Google")

Comment (barakplasma) [inline agent/browser/ie/pagetest/cdn.h]: My apologies, I should have gotten the formatting right the first time.

**Pull request merged**